docs/rules/prefer-object-spread.md

```diff
@@ -34,6 +35,9 @@ Examples of **correct** code for this rule:
 
 Object.assign(...foo);
 
+// Set a default value for "foo".
+{ foo: 'bar', ...baz }
```

Comment:
This line alone doesn't do anything useful. Maybe you meant to do an assignment?

```suggestion
const result = { foo: 'bar', ...baz };
```

Reply:
i agree it doesn't do anything useful by itself, but pretty much all the examples in here are the same.  doing `Object.assign({}, ...)` w/out assigning the return value is the same as writing `{...}`.